src/Esfa.Vacancy.Register.Api/Web.config

```diff
@@ -81,6 +81,7 @@
       <remove name="ApplicationInsightsWebTracking" />
       <add name="ApplicationInsightsWebTracking" type="Microsoft.ApplicationInsights.Web.ApplicationInsightsHttpModule, Microsoft.AI.Web" preCondition="managedHandler" />
     </modules>
+    <httpErrors existingResponse="PassThrough" />
```

Comment:
Could you put a comment on this PR explaining what this does for those that are not familiar.